src/mon/MonClient.h

```diff
@@ -213,14 +213,19 @@ class MonClient : public Dispatcher {
   void handle_subscribe_ack(MMonSubscribeAck* m);
 
   bool _sub_want(string what, version_t start, unsigned flags) {
-    if ((sub_new.count(what) == 0 &&
-	 sub_sent.count(what) &&
-	 sub_sent[what].start == start &&
-	 sub_sent[what].flags == flags) ||
-	(sub_new.count(what) &&
-	 sub_new[what].start == start &&
-	 sub_new[what].flags == flags))
+    auto sub_new_iter = sub_new.find(what);
```

Comment:
could simply use `sub` as the variable name, as no need to encode the type into the variable name.